platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformSchedulers.java

```diff
@@ -121,6 +125,21 @@ public static PlatformSchedulers create(@NonNull final PlatformContext context,
                 model.schedulerBuilder("stateSigner")
                         .withType(config.stateSignerSchedulerType())
                         .withUnhandledTaskCapacity(config.stateSignerUnhandledCapacity())
+                        .withMetricsBuilder(model.metricsBuilder().withUnhandledTaskMetricEnabled(true))
+                        .build()
+                        .cast(),
+                model.schedulerBuilder("applicationTransactionPrehandler")
+                        .withType(config.applicationTransactionPrehandlerSchedulerType())
+                        .withUnhandledTaskCapacity(config.applicationTransactionPrehandlerUnhandledCapacity())
+                        .withMetricsBuilder(model.metricsBuilder().withUnhandledTaskMetricEnabled(true))
+                        .withFlushingEnabled(true)
+                        .build()
+                        .cast(),
+                model.schedulerBuilder("stateSignatureCollector")
+                        .withType(config.systemTransactionPrehandlerSchedulerType())
+                        .withUnhandledTaskCapacity(config.systemTransactionPrehandlerUnhandledCapacity())
```

Comment:
In some places this is called `stateSignatureCollector`, in others `systemTransactionPrehandler`. Consider choosing one or the other

Reply:
Normalized to all use `stateSignatureCollector`. This was my original intent, I just was not very thorough with the refactor.